addons/mail/controllers/discuss.py

```diff
@@ -414,6 +414,9 @@ def channel_ping(self, channel_id, rtc_session_id=None, check_rtc_session_ids=No
     def mail_thread_data(self, thread_model, thread_id, request_list, **kwargs):
         res = {}
```

Comment:
I believe it would be nice to set hasAccess to true in the other case, so that if the user somehow gets access back he would be able to use the thread again.